src/pages/admin/main/components/donations-block/DonationsBlockForm.module.scss

```
@@ -0,0 +1,39 @@
+    margin-bottom: 40px;
+    align-items: center;
+}
+
+.text-section {
+    flex: 4;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+}
+
+.rich-text-custom :global(.editor-field),
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)